src/index.js

```diff
@@ -3,6 +3,7 @@ import todo from './todo-factory';
 import form from './form-load';
```

Comment:
### Mandatory
- [ ] The `webpack` is configured well to some extent. I do love that you went an extra mile to add the starting scripts but somehow forgot to add the `webpack-dev-server` dependency so it won't, run. Please fix this.

- [ ] We have some commented out code in the `stylesheet` that we might have forgotten to clean up. Kindly remove them. It helps keep our codebase cleaner and the entire project look professional. 




### Optional

- [ ] By disabling ESLint rules,  don't you think it rids you the opportunity to write much cleaner code and suppresses the potential warnings you’d be getting, thus missing out on the education that ESLint rules provide? 😟  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no-ignore-linters.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)).

- [ ] How about we had a delete button for the project if a user intends to delete a project?

- [ ] Lastly, could our forms used to create a project or a todo have validations in all fields? (consider this a stretch, thank you)